cmd/minishift/cmd/root.go

```diff
@@ -71,6 +74,15 @@ var RootCmd = &cobra.Command{
 
 		ensureConfigFileExists(constants.ConfigFile)
 
+		// Create MACHINE_NAME.json
+		jsonDataPath := filepath.Join(constants.Minipath, "machines", constants.MachineName+".json")
+		instanceConfig.Config, err = instanceConfig.NewInstanceConfig(jsonDataPath)
```

Comment:
@praveenkumar , `instanceConfig.Config` is already declared in `pkg/minishift/config/config.go` file. Remember ? so that we can access from other places.